java/code/src/com/redhat/rhn/frontend/strings/jsp/StringResource_en_US.xml

```diff
@@ -6157,6 +6157,15 @@ organization. You may grant or remove the organization administrator role in the
           <context context-type="sourcefile">/rhn/systems/details/packages/ScheduleRemoteCommand.do</context>
         </context-group>
       </trans-unit>
+      <trans-unit id="schedule.jsp.maint-window">
+        <source>Maintenance window:</source>
+      </trans-unit>
+      <trans-unit id="schedule.jsp.multiple_maintenance_schedules">
+        <source>There are multiple maintenance schedules for selected systems. Make sure that systems in the set use at most 1 maintenance schedule if you want to schedule by date or maintenance window.</source>
+      </trans-unit>
+      <trans-unit id="schedule.jsp.no_maintenance_windows">
+        <source>No upcoming maintenance windows</source>
+      </trans-unit>
```

Comment:
They need to be copied over to all other language files.
But we can also decide to do it later on the maintenance-windows branch

Reply:
Right, I noticed nobody does this, so I also decided not to :wink: